setup.cfg

```diff
@@ -39,15 +40,15 @@ console_scripts =
 
 [options.extras_require]
 doc =
+    Pallets-Sphinx-Themes
```

Comment:
(unrelated)

Oh, this is outdated. Should be `furo` instead.

If you want you can open a PR, otherwise I'll do it later.

Reply:
IMHO it's out of scope for this pr. Let's do it separately.

Reply:
Yes.